spec/malcontent/policy_spec.clj

```diff
@@ -43,6 +47,14 @@
                              "'allow-top-navigation'; "
                              "report-uri /some/report-uri"))
 
+(defn- policy=
+  "Checks two policy strings `expected` and `actual` are equal, irrespective of
+  the order."
+  [expected actual]
+  (letfn [(normalize [s]
+            (into #{} (map str/trim) (str/split s #";")))]
+    (= (normalize expected) (normalize actual))))
```

Comment:
I don't suppose we need to do anything to ensure deterministic results for fn's like `get-sources. I think the order there doesn't matter semantically.